content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -174,23 +174,22 @@ A `generator` block accepts the following fields:
 #### <inlinecode>binaryTargets</inlinecode> options
 
 The following tables list all supported operating systems with the name of platform to specify in [`binaryTargets`](/concepts/components/prisma-schema/generators#binary-targets).
+Unless specified otherwise, the default supported CPU architecture is x86_64.
```

Comment:
Do you want the line break in the rendered text here?
If so, we need to add a few spaces after the first paragraph.

Reply:
I didn't think that was necessary, but I can add a rendered `\n` if you feel that'd look better

Reply:
I am asking because you added a newline in the markdown, but that will be rendered directly after the previous paragraph: 
<img width="433" alt="image" src="https://user-images.githubusercontent.com/183673/217068754-8ce11a28-c024-4c6f-ad31-8ae4bc9817db.png">